### ISSUE TITLE
Can't start Linux miner

### ISSUE THREAD

**Issue by yao-ring** [trexminer/T-Rex#643]
Hello, I try to run the newest version of Linux miner on Centos7 without GUI, and the client return:
![image](https://user-images.githubusercontent.com/74333339/134768735-f555c73a-7e8b-4774-997f-cdb992cb7b45.png)

**Comment by yao-ring** [comment]
Is this a compatibility issue?

**Comment by trexminer** [comment]
Hi, could you post your t-rex cmd line arguments? Feel free to mask out your wallet address if you wish.
Also, at what point does it happen? Before miner prints out its own version, during mining, or other stage? You might've cut out potentially useful info in your screenshot.

**Comment by yao-ring** [comment]
Of course, here is my t-rex command: `./t-rex -a ethash -o stratum+tcp://cn.ezil.me:5555 -u 0xc52AC8bdABcdef1d0afe3FF9e9949D0abcd.zil1m94phupcldkefpatmnr0cphr8wugq053dxf--extra-dag-epoch 0`.
And I think this bug is throwed at begining, because I can not manage to check the version info.
![image](https://user-images.githubusercontent.com/74333339/134789185-6e54da97-dc40-42ed-9651-034bcff23644.png)

**Comment by trexminer** [comment]
Ok, thanks. Will look into it.

**Comment by trexminer** [comment]
What's your kernel version by the way?

**Comment by yao-ring** [comment]
Centos 7 server, `Linux 3.10.0-514.el7.x86_64 x86_64`
